assets/models/system/openai-whisper-large/description.md

```diff
@@ -26,10 +26,7 @@ Batch |<a href="https://aka.ms/azureml-infer-batch-sdk-asr" target="_blank">asr-
 ```json
 [
     {
-        "text": "This is a bright day."
-    },
-    {
-        "text": "This is a bright day."
+        "text": "Since that day, he had never been heard of. In vain, Marguerite dismissed her guests, changed her way of life. The Duke was not to be heard of. I was the gainer in so."
```

Comment:
thanks Tanmay. Please also do a version up for whisper model for successful publish. Thanks

Reply:
Done